src/utils.ts

```diff
@@ -1,7 +1,30 @@
-// Util to normalize windows paths to posix
-export function normalizeWindowsPath (input: string = '') {
-  if (!input.includes('\\')) {
-    return input
+import { basename } from 'pathe'
```

Comment:
Please use relative import when possible for library source references.

Reply:
If we use relative import then rollup will chunk the build, and I assume that is not what we want.

Reply:
It is safe bundle saving behavior of unbuilding actually and we use it in serveral packages.

Reply:
Ah. But it's not that helpful with regard to vercel/nft behaviour. It effectively preserves entire package in traced output, when the purpose (I _thought_) of the `/utils` subpath was to split out utils from main chunk.

Reply:
I guess rollup would create a _common_ chunk. Which nft has to track. `/` subpath bundling (index + common chunk). It is same size overhead.

Reply:
Nope, it's not relevant.

Yes. Rollup creates a common chunk with all utilities + core in it.